libparsec/crates/client_connection/src/lib.rs

```diff
@@ -1,5 +1,6 @@
 // Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
 
+mod account;
```

Comment:
```suggestion
mod annonymous_account_cmds;
mod authenticated_account_cmds;
```
I think we shouldn't put those new families in an `account` module:
- Going by this logic, we should have a "non account" module containing the other families...
- There is nothing shared between the account families anyway